ext/intl/tests/rangeformatter/basic.phpt

```diff
@@ -0,0 +1,395 @@
+--TEST--
+Basic test for IntlNumberRangeFormatter
+--EXTENSIONS--
+intl
+--SKIPIF--
+<?php
+if (version_compare(INTL_ICU_VERSION, '63.0') < 0) {
+    die('skip for ICU < 63.0');
+}
+?>
+--FILE--
+<?php
+
+$languages = ['en_US', 'RO', 'JA'];
+
+$identityFallbacks = [
+    'IDENTITY_FALLBACK_SINGLE_VALUE' => IntlNumberRangeFormatter::IDENTITY_FALLBACK_SINGLE_VALUE, 
+    'IDENTITY_FALLBACK_APPROXIMATELY_OR_SINGLE_VALUE' => IntlNumberRangeFormatter::IDENTITY_FALLBACK_APPROXIMATELY_OR_SINGLE_VALUE,
+    'IDENTITY_FALLBACK_APPROXIMATELY' => IntlNumberRangeFormatter::IDENTITY_FALLBACK_APPROXIMATELY,
+    'IDENTITY_FALLBACK_RANGE' => IntlNumberRangeFormatter::IDENTITY_FALLBACK_RANGE
+];
+
+$collapses = [
+    'COLLAPSE_AUTO' => IntlNumberRangeFormatter::COLLAPSE_AUTO,
+    'COLLAPSE_NONE' => IntlNumberRangeFormatter::COLLAPSE_NONE,
+    'COLLAPSE_UNIT' => IntlNumberRangeFormatter::COLLAPSE_UNIT,
+    'COLLAPSE_ALL' => IntlNumberRangeFormatter::COLLAPSE_ALL
+];
+
+foreach ($languages as $language) {
+    foreach ($identityFallbacks as $iName => $identityFallback) {
+        foreach ($collapses as $cName => $collapse) {
+            echo PHP_EOL . $language . ' - ' . $cName . ' - ' . $iName . PHP_EOL;
+            $nrf = IntlNumberRangeFormatter::createFromSkeleton(
+                'measure-unit/length-meter', 
+                $language, 
+                $collapse, 
+                $identityFallback
+            );
+
+            var_dump($nrf->format(1.1, 2.2));
+            var_dump($nrf->format(100, 200));
+            var_dump($nrf->format(-5, 5));
+            var_dump($nrf->format(5, 5));
+            var_dump($nrf->format(5.0001, 5.0001));
```

Comment:
can you add cases where first op is of zend_long, second a double and vice versa ?

Reply:
done :)